### ISSUE TITLE
Unpuncture lungs

### ISSUE THREAD

**Issue by StealthDrifter** [studio-minus/ppg-bugs-and-requests#706]
**Feature description**
A way to heal punctured lungs maybe as an item
could also just be a context menu button

**Feature purpose**
So you don't have to respawn a human or so you can roleplay hospitals

**Comment by mestiez** [comment]
Yes I want this too, but I'm not sure what kind of item would make sense in this situation.

**Comment by D3stStrut** [comment]
I could see some surgical tools or a medical expansion as a fix. Plus, most players would want that.

**Comment by Cboyjet4** [comment]
> Yes I want this too, but I'm not sure what kind of item would make sense in this situation.

There's an IRL object for it, but it'd probably be a bit too specific, punctured lungs while a person is still living is pretty uncommon in any circumstance in this game, so I'd just say dedicating/renaming the water breathing syringe for fixing lungs could give it more than one purpose.

**Comment by Cboyjet4** [comment]
> I could see some surgical tools or a medical expansion as a fix. Plus, most players would want that.

Surgical tools would be a very odd addition and they would probably not work, specific tools for doing things might, I've always enjoyed playing doctor, and just using a life syringe isn't enaging or fun for reviving humans.

**Comment by Gigi-7688** [comment]
Yes, I would like it very much.

**Comment by vda8** [comment]
> Yes I want this too, but I'm not sure what kind of item would make sense in this situation.

any sort of healing items like:
mending syringe.
reconstructor.
health syringe.
Etc.
